aiogram/types/dice.py

```diff
@@ -17,3 +17,5 @@ class DiceEmoji:
     DICE = '🎲'
     DART = '🎯'
     BASKETBALL = '🏀'
+    SOCCER = '⚽️'
```

Comment:
https://core.telegram.org/bots/api-changelog#November-4-2020
> Supported the new **football** and slot machine animations for the random dice.

Also this two emojis are different: '⚽', '⚽️'. You should use the first, because it used in api responses.

```suggestion
    FOOTBALL = '⚽'
```